src/main/java/org/folio/rest/impl/AddressTypeAPI.java

```diff
@@ -168,7 +168,7 @@ public void getAddresstypesByAddresstypeId(String addresstypeId, String lang,
       try {
         Criterion criterion = new Criterion(
           new Criteria().addField(ID_FIELD_NAME).
-            setJSONB(false).setOperation("=").setValue("'" + addresstypeId + "'"));
+            setJSONB(false).setOperation("=").setVal(addresstypeId));
```

Comment:
Can you replace the complete method by a PgUtil.getById invokation so that this duplicated code gets removed?

    @Override
    public void getAddresstypesByAddresstypeId(String addresstypeId, String lang,
          Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler,
          Context vertxContext) {
      PgUtil.getById(ADDRESS_TYPE_TABLE, AddressType.class, addresstypeId,
          okapiHeaders, vertxContext,  GetAddresstypesByAddresstypeIdResponse.class, asyncResultHandler);
    }